tests/test_ssa_value.py

```diff
@@ -37,14 +37,21 @@ def test_var_mixed_builder():
         _ = b.get(op)
 
 
-@pytest.mark.parametrize("name", ["a", "test", "1", "12", "-2", "test_123"])
+@pytest.mark.parametrize("name", ["a", "test", "-2", "test_123", "kebab-case-name"])
 def test_ssa_value_name_hints(name):
-    """
-    The rewriter assumes, that ssa value name hints (their .name field) does not end in
-    a numeric value. If it does, it will generate broken rewrites that potentially assign
-    twice to an SSA value.
+    r"""
```

Comment:
ultra nit: This should probably not be a documentation comment, but just a normal comment.
Because documentation comment are attached to a particular function, while here you are referencing "the next three text".